### ISSUE TITLE
NullPointerException in LocationService.onStart()

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#201]
```
GTalkSMS v3.0

java.lang.RuntimeException: Unable to start service 
com.googlecode.gtalksms.LocationService@405284b8 with null: 
java.lang.NullPointerException
at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:2338)
at android.app.ActivityThread.access$2800(ActivityThread.java:132)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1106)
at android.os.Handler.dispatchMessage(Handler.java:99)
at android.os.Looper.loop(Looper.java:143)
at android.app.ActivityThread.main(ActivityThread.java:4196)
at java.lang.reflect.Method.invokeNative(Native Method)
at java.lang.reflect.Method.invoke(Method.java:507)
at 
com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:839)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:597)
at dalvik.system.NativeStart.main(Native Method)
Caused by: java.lang.NullPointerException
at com.googlecode.gtalksms.LocationService.onStart(LocationService.java:115)
at android.app.Service.onStartCommand(Service.java:428)
at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:2321)
... 10 more
```

Original issue reported on code.google.com by `Florent....@gmail.com` on 22 Jul 2011 at 9:45

**Comment by GoogleCodeExporter** [comment]
```
Seems like the intent or its action was null: 
http://code.google.com/p/gtalksms/source/browse/src/com/googlecode/gtalksms/Loca
tionService.java?name=v3.0#115
```

Original comment by `fschm...@gmail.com` on 26 Aug 2011 at 4:23

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 31 Aug 2011 at 10:59
- Changed title: **NullPointerException in LocationService.onStart()**
- Added labels: **FromMarket**

**Comment by GoogleCodeExporter** [comment]
```
No reports for this in a while, lowering priority.
```

Original comment by `fschm...@gmail.com` on 12 Apr 2012 at 1:24
- Added labels: **Priority-Low**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
With GTalkSMS 4.2:

java.lang.RuntimeException: Unable to start service 
com.googlecode.gtalksms.LocationService@41cb4170 with null: 
java.lang.NullPointerException
at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:2425)
at android.app.ActivityThread.access$1900(ActivityThread.java:127)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1239)
at android.os.Handler.dispatchMessage(Handler.java:99)
at android.os.Looper.loop(Looper.java:137)
at android.app.ActivityThread.main(ActivityThread.java:4524)
at java.lang.reflect.Method.invokeNative(Native Method)
at java.lang.reflect.Method.invoke(Method.java:511)
at 
com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:809)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:576)
at dalvik.system.NativeStart.main(Native Method)
Caused by: java.lang.NullPointerException
at com.googlecode.gtalksms.LocationService.onStart(LocationService.java:122)
at android.app.Service.onStartCommand(Service.java:438)
at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:2408)
... 10 more
```

Original comment by `fschm...@gmail.com` on 11 Nov 2012 at 11:09

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision bc77567c3519.
```

Original comment by `fschm...@gmail.com` on 11 Nov 2012 at 11:16
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed with 4.3 release
```

Original comment by `fschm...@gmail.com` on 26 Jan 2013 at 5:45
- Changed state: **Deployed**
